pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go

```diff
@@ -52,6 +52,12 @@ type Secret struct {
 	MountPath string `json:"mountPath"`
 }
 
+type HostFile struct {
+	Name      string `json:"name"`
+	MountPath string `json:"mountPath"`
+    NodePath  string `json:"nodePath"`
```

Comment:
Some indentation issue is here also in line: 85

Reply:
fixed!